<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1090</width>
    <height>806</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="backgroundLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1071</width>
      <height>791</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="mapLabel">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>110</y>
      <width>301</width>
      <height>221</height>
     </rect>
    </property>
    <property name="text">
     <string>map</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="seattleButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>360</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Seattle</string>
    </property>
   </widget>
   <widget class="QPushButton" name="japanButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>360</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Japan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="londonButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>360</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>London</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="clockGroupBox">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>410</y>
      <width>341</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLCDNumber" name="hourLcd">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>101</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="digitCount">
      <number>2</number>
     </property>
     <property name="value" stdset="0">
      <double>11.000000000000000</double>
     </property>
    </widget>
    <widget class="QLCDNumber" name="minuteLcd">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>20</y>
       <width>101</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="digitCount">
      <number>2</number>
     </property>
     <property name="value" stdset="0">
      <double>11.000000000000000</double>
     </property>
    </widget>
    <widget class="QLCDNumber" name="secondLcd">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>40</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="digitCount">
      <number>2</number>
     </property>
     <property name="value" stdset="0">
      <double>11.000000000000000</double>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="tempLabel">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>560</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>weather temp</string>
    </property>
   </widget>
   <widget class="QLabel" name="conditionLabel">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>560</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>weather condition</string>
    </property>
   </widget>
   <widget class="QLabel" name="iconLabel">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>610</y>
      <width>91</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>weather icon</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="zipCodeEdit">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>70</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>98119</string>
    </property>
   </widget>
   <widget class="QPushButton" name="locationButton">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>70</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Get Location</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1090</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
